workflows/check_commit_message_for_tickets.yml

```diff
@@ -0,0 +1,92 @@
+name: Check Commit Message
+
+on:
+  pull_request:
+    types: [opened, synchronize, edited, reopened]
+
+jobs:
+  check-commit-message:
+    runs-on: ubuntu-latest
+    env:
+      REGEX_PATTERN: '^(?i)(ref|refs|reference|references|res|resolve|resolves)[ \t]*:[ \t]*(gh-|\#|\!|[A-Za-z]+-)\d+\s*$'
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install jq
+        run: sudo apt-get install jq
```

Comment:
`jq` should already [be installed](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md)

Reply:
Agreed, it should. If it is, this becomes a noop effectively.